### DISCUSSION TITLE
Bulk entry sectioning

### DISCUSSION THREAD

**Discussion by jtruland** [mealie-recipes/mealie#2898] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Bulk entry of copy/paste text from word documents is tedious, having to go back to add/enter section headers separately is even more so.

### Please provide a concise description of the feature that would resolve your issue.

Add some type of text/markup code parsing to add/define section headers in the bulk add editor.

### Please consider and list out some caveats or tradeoffs made in your design decision

N/A

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by figadore** [comment]
I'd love for this to exist, I'm trying to migrate from a large unwieldy document, but it's proven very tedious to import manually

Even if I had to copy/paste a recipe at a time and have it parse the same way a url is parsed, that would be less tedious.
